README.md

```diff
@@ -2,6 +2,8 @@
 
  [![Build Status](https://travis-ci.org/OpenSourceHelpCommunity/OpenSourceHelpCommunity.github.io.svg?branch=develop)](https://travis-ci.org/OpenSourceHelpCommunity/OpenSourceHelpCommunity.github.io)
 
+[![Coverage Status](https://coveralls.io/repos/github/OpenSourceHelpCommunity/OpenSourceHelpCommunity.github.io/badge.svg?branch=develop)](https://coveralls.io/github/OpenSourceHelpCommunity/OpenSourceHelpCommunity.github.io?branch=develop)
+
```

Comment:
@cmdrwebdev First of all congratulations!! :tada: :tada: You have made your first PR and it needs minor changes. You have to push commit again in order to reflect new changes.